incuna-sass/legacy/_box-sizing.sass

```diff
@@ -1,14 +1,14 @@
-%border-box
+%legacy-border-box
```

Comment:
but we don't have modern ones of the same type right? or is it because there could be in bourbon?